com.ibm.wala.cast.java/build.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'java'
+    
+repositories {
```

Comment:
can we move these `repositories` declarations to a top-level `build.gradle` file?